### ISSUE TITLE
Install Apache Spark

### ISSUE THREAD

**Issue by enragedginger** [sequenceiq/docker-ambari#121]
I have a cluster up and running using the docker-ambari image and accompanying setup scripts. Now that Ambari is running, how do I install Apache Spark on my cluster? Is there an easy way to do this via the Ambari Shell? Or is there some way to incorporate the [Docker Spark image](https://github.com/sequenceiq/docker-spark) into this setup?

**Comment by biharitomi** [comment]
Hi,

The [https://github.com/sequenceiq/docker-spark](Docker Spark image) could work, but you could also add Spark as a new service on the existing Ambari's UI. As the [doc](https://docs.hortonworks.com/HDPDocuments/Ambari-2.2.1.0/bk_Ambari_Users_Guide/content/_adding_a_service.html) describes:
`To add a service, select Actions > Add Service, then complete the following procedure using the Add Service Wizard.`

Br,
Tamas

**Comment by enragedginger** [comment]
Well, now I feel like an idiot. I knew that I could install Spark via Ambari's UI, but I didn't know that Ambari's UI was available to me with this setup. I had tried connecting to the UI on each of the available ports that the Docker image was exposing, but must have missed something. Which port is it listening on?

**Comment by biharitomi** [comment]
It is listening on the `8080` port of the Ambari server container.
You should inspect the ip of the Ambari server, if you use our Ambari functions, it is easy to do that by using `get-ambari-server-ip` function.

**Comment by enragedginger** [comment]
Okay, I managed to get it working. Originally, I was following the instructions from [this blog post](http://blog.sequenceiq.com/blog/2014/12/04/multinode-ambari-1-7-0/) which doesn't mention that the UI is available. I did some poking around and tried hitting what I thought was the Ambari server's IP on port 8080, but must have had the wrong IP.

Regardless, it's working now. Thanks for the assistance!
